2-python-fundamentals-i.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
_Commented on notebook `2-python-fundamentals-i.ipynb` Cell 141 Line 7_
```markdown
the syntax `for i in range(n):` will execute the indented code `print (i**2)` a total of `n` times, with `i = 0, 1, 2, ... n-1`

#### An important note on python syntax

Indents in python code matter, unlike in C, C++, or Java. Indents are actually a form of syntax in python. 

Python requires indenting loops and conditionals (we will get to these a bit later). Indenting can be achieved using either **one <kbd>TAB</kbd>** or **four <kbd>SPACEBAR</kbd>** keystrokes. There is a lively debate over which of these is preferred. Text editors now make it easy to switch between tabs an spaces - the important thing is to remain consistent. 
```
At risk of stirring up the most infamously low stakes python debate... PEP8 is spaces.

Maybe advice to use spaces, and inform that when you press the TAB key in VSCode (and most python editors), it'll write out 4 spaces instead.